flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/JsonFunctionsITCase.java

```diff
@@ -54,6 +77,7 @@
         testCases.add(jsonValue());
         testCases.addAll(isJson());
         testCases.addAll(jsonQuery());
+        testCases.addAll(jsonObjectSpec());
```

Comment:
nit: why `Spec` here?

Reply:
Because `jsonObject` is also the name of the API method, creating a clash. I'll just rename them all for consistency.